SCClassLibrary/Common/GUI/PlusGUI/Control/ServerPlusGUI.sc

```diff
@@ -160,14 +167,24 @@ ServerTreeView {
 			}
 		}, '/g_queryTree.reply', this.server.addr).fix;
 
-		updateFunc = {
-			updater = fork {
+		updater = {
+			fork {
 				loop {
+					defer {
+						if (this.server.serverIsRunning) {
+							treeViewStatus.string_(" STARTED: current server tree");
+						} {
+							treeViewStatus.string_(" STOPPED: last updated server tree")
+						}
+					};
 					this.server.sendMsg("/g_queryTree", 0, 0);
 					interval.wait;
 				}
 			}
 		};
+		updateFunc = {
+			updater.()
```

Comment:
`updater.value;` to align styistically with line 188.

Reply:
ok! I have done!